packages/suite/src/utils/suite/stake.ts

```diff
@@ -460,3 +472,75 @@ export const getStakeTxGasLimit = async ({
         };
     }
 };
+
+export const getUnstakingPeriodInDays = (validatorWithdrawTimeInSeconds?: number) => {
+    if (!validatorWithdrawTimeInSeconds) {
+        return UNSTAKING_ETH_PERIOD;
+    }
+
+    return Math.round(validatorWithdrawTimeInSeconds / 60 / 60 / 24);
+};
+
+const secondsToDays = (seconds: number) => Math.round(seconds / 60 / 60 / 24);
+
+export const getDaysToAddToPool = (
+    stakeTxs: WalletAccountTransaction[],
+    validatorsQueue?: ValidatorsQueue,
+) => {
+    if (
+        validatorsQueue?.validatorAddingDelay === undefined ||
+        validatorsQueue?.validatorActivationTime === undefined
+    ) {
+        return undefined;
+    }
+
+    const lastTx = stakeTxs[0];
+
+    if (!lastTx?.blockTime) return 1;
+
+    const sevenDays = 7 * 24 * 60 * 60;
```

Comment:
This can be extracted to a const out of the function scope:

Also please lets name it `const SEVEN_DAYS_IN_SECONDS`

Reply:
01fcab5e90501501990bc3dfeed93ed268d7e0e3